source/weeknotes/index.html.md.erb

```diff
@@ -0,0 +1,33 @@
+---
+layout: singlepage
+blog: weeknotes
+---
+
+# Weeknotes
+
+We send out an update on our work each week. Here’s the latest, and there’s
+a <a href="#previous">list of the older ones below</a>.
+
+---
+
+<h2>Latest: <%= blog.articles.first.title %></h2>
+<%= blog.articles.first.body %>
```

Comment:
It'd be nice if the latest had the date, like the latest blog does

Reply:
Addressed here: https://github.com/fluidkeys/www.fluidkeys.com/pull/51